pkg/cluster/deploystorage.go

```diff
@@ -248,6 +240,18 @@ func (m *manager) attachNSGsAndPatch(ctx context.Context) error {
 		}
 	}
 
+	return nil
+}
+
+func (m *manager) patchKubeconfig(ctx context.Context) error {
```

Comment:
`s/patchKubeconfig/generateKubeconfigs`

Reply:
And maybe move to separate file `kubeconfig.go` as separate commit outside code change

Reply:
`generateKubeoConfig(s)` can be seen as a function that does all the work calling the other `generateKubeConfig` in `kubeconfig.go`  (so when it's in the install step, it will "generateKubeConfig(s) as a way to understand that all kubeconfigs should be generated during this step.